app/views/shared/_tasks_report_list.html.erb

```diff
@@ -0,0 +1,29 @@
+<table class="table table-striped table bordered">
+  <thead>
+    <tr>
+      <th>Project Name</th>
+      <th>Name</th>
+      <th>Start Date</th>
+      <th>Expected End Date</th>
+      <th>Assigned To</th>
+      <th>Status</th>
+    </tr>
+  </thead>
+  <tbody>
+    <% @tasks.each do |task|%>
+      <tr>
+        <td><%= task.project.name %></td>
+        <td><%= task.name %></td>
+        <td><%= task.start_date.to_date %></td>
+        <td><%= task.expected_end_date.to_date %></td>
```

Comment:
could use task.start_date.to_formatted_s(:short), it gives a  nice short formatted form of date